config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'upload/' => 'upload#index', as: 'upload'
```

Comment:
Everything for a particular model should be in the same controller. index, show, create, edit, delete, update. http://guides.rubyonrails.org/action_controller_overview.html